Final Project Submission - Understanding Gender in Dubliners with CLA

fcr3 opened the pull request:
Gender-centered literary criticisms about James Joyce's Dubliners are few and far between, and new perspectives are needed to shed light on gender relations displayed in Dubliners. Taking this challenge on, I present an understanding of gender in Dubliners through computational literary analysis.

JonathanReeve commented:
Could you modify this pull request to only include your project notebook? Thanks!

fcr3 commented:
> Could you modify this pull request to only include your project notebook? Thanks!

I have added a new commit. In order to make sure that the code runs correct, I have included a part of the import step that installs gdown, imports gdown, and downloads the james joyce photo.

JonathanReeve commented:
Ok great. Merging this. But if you just want to include a photo in your notebook, all you have to do is include its URL in markdown. So `![Description of the photo](http://url-of-the-photo)` would do the trick.

fcr3 commented:
Thank you! Also I wanted to call out that I’ve added two dependencies: wordcloud and gdown. These dependencies might not be installed on your system. I also added the Readings directory in a place one directory up from the root of this project. In case you get file error issues, I think you can just add an extra ../ to that file path to point to the perspective of the root.
